### PR TITLE
support read github file raw content

### PR DESCRIPTION
**Pull request by shireenf-ibm**
I made a change in an existing test, hoping it is okay

### CODE REVIEW

**Review by zivnevo — CHANGES_REQUESTED** [review]
a few minor comments

File: src/baseline_rule.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import json
 import base64
 from urllib import request
+from urllib.request import urlopen
```
[the remark this answers is not in the document]
Comment (shireenf-ibm) [reply src/baseline_rule.py:13]: Right

File: tests/testBaselineRuleToNetPol.py [hunk 2]
```diff
@@ -175,7 +175,7 @@ def compare_strings(expected, actual):
 
 
 def get_rule_url(rule_name):
-    return f'https://github.com/np-guard/baseline-rules/blob/master/examples/{rule_name}.yaml'
+    return f'https://raw.githubusercontent.com/np-guard/baseline-rules/master/examples/{rule_name}.yaml'
```
[the remark this answers is not in the document]
Comment (shireenf-ibm) [reply tests/testBaselineRuleToNetPol.py:178]: We have the same method with non-raw urls in the testBaselineRuleToGlobalNetPol.py test file.

[the remark this answers is not in the document]
Comment (zivnevo) [reply tests/testBaselineRuleToNetPol.py:178]: Ah sorry, I forgot about the new `testBaselineRuleToGlobalNetPol.py`.
It's ok then.

**Review by zivnevo — APPROVED** [review]
ok

**Pull request merged**